action.yml

```
@@ -1,8 +1,8 @@
-name: Nix Installer
+name: Determinate Nix Installer
 branding:
   icon: "box"
   color: "purple"
-description: Install Nix
+description: Install Nix with the Determinate Nix Installer, with Flakes enabled by default.
```

```suggestion
description: Install Nix with the Determinate Nix Installer: https://github.com/DeterminateSystems/nix-installer.
```

I don't know where this description shows up, but if it supports markdown, this would be better:

```suggestion
description: Install Nix with the [Determinate Nix Installer](https://github.com/DeterminateSystems/nix-installer).
```

---

Think it shows up in the "Marketplace" entry which I don't think anyone uses.

---

```suggestion
description: Install Nix with the Determinate Nix Installer. See: https://github.com/DeterminateSystems/nix-installer
```